{{cookiecutter.project_name}}/01-documentation/README.md

```diff
@@ -0,0 +1,3 @@
+# This is the documentation of {{cookiecutter.project_name}}
+
+You can use the [Mardkown syntax](https://www.markdownguide.org/basic-syntax/)  to organize your documentation.
```

Comment:
```suggestion
You can use the [Markdown syntax](https://www.markdownguide.org/basic-syntax/)  to organize your documentation.
```

Correction also needed for the other files

It would be nice to have the TODO statement for here too